### DISCUSSION TITLE
Elasticsearch with authentication

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#12463] [Q&A / Support] [answered]
Hello,

We are using the elastic search in our one Application, sometimes there is generated unnecessary entries into ES indexing. so we have active some security rules and set the username & password.
So now anyone can not able create or delete an index without authentication. Now the issue is how to configure this username & password into the elastic-search.yml file because we can not able to access the frontend also without the authentication.

error on frontend:
![image](https://user-images.githubusercontent.com/52573912/174831992-7244fe99-8faf-4438-80ba-7606409cef86.png)

ES Config:
![image](https://user-images.githubusercontent.com/52573912/174832283-40caeaec-dc1b-4375-8c40-8396ec465e01.png)

So is there any way how to add the username & password above into the code?

**Comment by podarcis** [comment, marked as answer]
One way is to provide your env var `ES_HOST` in the form `http://username:password@elasticsearch-host:9200`.

**Reply by ghost** [reply]
Thanks, @podarcis it works for me.
